Extension/package.json

```diff
@@ -282,2560 +282,2602 @@
-        },
-        "C_Cpp.debugShortcut": {
-          "type": "boolean",
-          "default": true,
-          "description": "%c_cpp.configuration.debugShortcut.description%",
-          "scope": "resource"
-        },
-        "C_Cpp.legacyCompilerArgsBehavior": {
-          "type": "boolean",
-          "default": false,
-          "markdownDescription": "%c_cpp.configuration.legacyCompilerArgsBehavior.markdownDescription%",
-          "scope": "resource"
-        },
-        "C_Cpp.sshTargetsView": {
-          "type": "string",
-          "enum": [
-            "enabled",
-            "disabled",
-            "default"
-          ],
-          "default": "default",
-          "description": "%c_cpp.configuration.sshTargetsView.description%",
-          "scope": "window"
+          "C_Cpp.loggingLevel": {
+            "type": "string",
+            "enum": [
+              "None",
+              "Error",
+              "Warning",
+              "Information",
+              "Debug"
+            ],
+            "default": "Error",
+            "markdownDescription": "%c_cpp.configuration.loggingLevel.markdownDescription%",
+            "scope": "window"
+          },
+          "C_Cpp.autoAddFileAssociations": {
+            "type": "boolean",
+            "default": true,
+            "markdownDescription": "%c_cpp.configuration.autoAddFileAssociations.markdownDescription%",
+            "scope": "window"
+          },
+          "C_Cpp.preferredPathSeparator": {
+            "type": "string",
+            "enum": [
+              "Forward Slash",
+              "Backslash"
+            ],
+            "default": "Forward Slash",
+            "markdownDescription": "%c_cpp.configuration.preferredPathSeparator.markdownDescription%",
+            "scope": "machine-overridable"
+          },
+          "C_Cpp.updateChannel": {
+            "type": "string",
+            "enum": [
+              "Default",
+              "Insiders"
+            ],
+            "default": "Default",
+            "markdownDescription": "%c_cpp.configuration.updateChannel.markdownDescription%",
+            "scope": "application",
+            "deprecationMessage": "%c_cpp.configuration.updateChannel.deprecationMessage%"
+          },
+          "C_Cpp.experimentalFeatures": {
+            "type": "string",
+            "enum": [
+              "enabled",
+              "disabled"
+            ],
+            "default": "disabled",
+            "description": "%c_cpp.configuration.experimentalFeatures.description%",
+            "scope": "window"
+          },
+          "C_Cpp.caseSensitiveFileSupport": {
+            "type": "string",
+            "enum": [
+              "default",
+              "enabled"
+            ],
+            "default": "default",
+            "markdownDescription": "%c_cpp.configuration.caseSensitiveFileSupport.markdownDescription%",
+            "scope": "window"
+          },
+          "C_Cpp.addNodeAddonIncludePaths": {
```

Comment:
This setting applies to IntelliSense include paths so it seems like it should be in the IntelliSense section?

Reply:
Actually, I think "Workspace Configuration" is better. That's where the other include path setting is.

Reply:
Yeah, I'm confused as to what the difference is, i.e. how is user supposed to know which section the setting is in? I'm worried users might end up just having to look in both sections.

Reply:
I think the sub headers are more for browsing.  I would expect most people to use the search bar to find the settings they want.  I agree that there are going to be some settings that could potentially fit in more than one category.  If we get feedback that settings are too hard to find, we can re-evaluate.

Reply:
Yeah, but what's the guideline as to which section a user should browse for an IntelliSense setting? Seems like they have to look in both IntelliSense and Workspace Configuration, so they might as well be combined. It only makes sense to split something into a different section if it reduces the amount of browsing that might be necessary.

Also, as a developer, I'm not going to know which section to put any new setting I create.

Reply:
I believe that looking in 2 sections is better than looking in a giant flat list that includes all of the extension's settings though. These are essentially filters.

If you need help assigning a new setting to a section you can ask me and we can talk about it.

Reply:
If users have to look in both sections, then it seems worse.

In particular, the "errorSquiggles" setting seems like it's a pretty solidly IntelliSense setting to me and isn't related to Workspace Configuration, right?

Reply:
Yeah, I'd prefer "IntelliSense" and "Configuration Defaults" (or some variation on that naming) with just the C_Cpp.default settings or maybe "Configuration Management" if you wanted to add other settings directly related to c_cpp_properties.json configuration. Or any other section grouping that is easy to understand.

A "Resource Management" setting (or whatever it's named) sounds good to me.

Reply:
@bobbrow ping

Reply:
Sounds good.  I'll merge the IntelliSense sections and have a new one for resource management.